### DISCUSSION TITLE
External links only?

### DISCUSSION THREAD

**Discussion by cinderblockgames** [go-shiori/shiori#697] [Q&A] [answered]
Is there a way to prevent bookmarks from generating local versions and linking to them?  I just want to see the thumbnail and always go to the external site.  (Looking for somewhere to dump my 100+ open tabs so I don't have to have them in my browser waiting for me to come back to them.)

**Comment by Monirzadeh** [comment]
you can stop create archive by default we have that in archive.
i think you need an option to not create reader mode(not archive) or option to open external link instead of reader mode right?

**Reply by cinderblockgames** [reply]
Yes, exactly.

**Reply by Monirzadeh** [reply]
so create reader mode is not problem for you (we use that for generate ebook) just option to open external link.
for now we change settings code  a little bit  to easier add setting later. maybe i can add that after #639 merge. not promise but i consider that.

**Comment by fmartingr** [comment, marked as answer]
Adding up to @Monirzadeh comments, by default all bookmarks retrieve the readable version of the articles. If you click on the title you will be taken to the readable version. Alternatively, you can click on the bottom-left hostname of the bookmark to go directly to the bookmark URL. Hope that's helpful.

I see how readable versions may not be useful under some circumstances but I assume it was made that way so metadata for the bookmark was extracted automatically (title, excerpt). I need to sit down on this and think what is the best course of action, but there's a bunch of things pending yet before we can commit to more UX changes, I have some ideas and will try to render them into mockups somewhere.

**Reply by cinderblockgames** [reply]
> Alternatively, you can click on the bottom-left hostname of the bookmark to go directly to the bookmark URL. Hope that's helpful.

Thank you! That is exactly what I needed to know.
